src/main/java/com/kiwigrid/helm/maven/plugin/InitMojo.java

```diff
@@ -145,4 +140,14 @@ public boolean isSkipRefresh() {
 	public void setSkipRefresh(boolean skipRefresh) {
 		this.skipRefresh = skipRefresh;
 	}
+
+	public void initHelmClient() throws MojoExecutionException {
```

Comment:
pls. make this a `private` method
Once this done, I'm happy to merge :)

Reply:
Done